custom/conf/app.example.ini

```diff
@@ -1046,3 +1046,29 @@ QUEUE_CONN_STR = "addrs=127.0.0.1:6379 db=0"
 MAX_ATTEMPTS = 3
 ; Backoff time per http/https request retry (seconds)
 RETRY_BACKOFF = 3
+
+; default storage for attachments, lfs and avatars
+[storage]
+; storage type
+STORE_TYPE = local
+
+; customize storage
+[storage.my_minio]
+STORE_TYPE = minio
+; Minio endpoint to connect only available when STORE_TYPE is `minio`
+MINIO_ENDPOINT = localhost:9000
+; Minio accessKeyID to connect only available when STORE_TYPE is `minio`
+MINIO_ACCESS_KEY_ID =
+; Minio secretAccessKey to connect only available when STORE_TYPE is `minio`
+MINIO_SECRET_ACCESS_KEY =
+; Minio bucket to store the attachments only available when STORE_TYPE is `minio`
+MINIO_BUCKET = gitea
+; Minio location to create bucket only available when STORE_TYPE is `minio`
+MINIO_LOCATION = us-east-1
+; Minio enabled ssl only available when STORE_TYPE is `minio`
+MINIO_USE_SSL = false
```

Comment:
Can you comment those out? Some people might copy the config verbatim, so it's better to not have such active examples in it.